collect-atomic-banks-with-elements/api.js

```diff
@@ -10,11 +10,11 @@ const router = express.Router();
 bt.init(SERVER_KEY);
 
 const executeSpreedlyReactor = async ({ bankTokenId, name }) => {
-  const reactionToken = await bt.atomicBanks.react(bankTokenId, {
-    reactorId: REACTOR_ID,
-    requestParameters: {
-      ACCOUNT_OWNER_FULL_NAME: name,
-    },
+  const reactionToken = await bt.reactors.react(REACTOR_ID, {
```

Comment:
I would rename the variable to `reactorResponse` - we don't return a token from reactors anymore

Reply:
Or just destruct it and grab the raw.
```
const { raw } = bt (...)
```